connectionpool_systems_test.go

```diff
@@ -0,0 +1,266 @@
+// Copyright (c) 2015 The gocql Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+// +build conn_pool
+
+package gocql
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os/exec"
+	"strconv"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+)
+
+// connection pool behavior test when nodes are removed from the cluster
+// to run this test, see connectionpool_systems_test.sh
+
+var (
```

Comment:
ah I see what its doing now. Its expecting that its being ran in as part of a build against `ccm`. Our test harness is becoming rather complex now :)

It would be nice if the flags were only defined once though.